pkg/util/metrics.go

```diff
@@ -26,3 +29,10 @@ func NewRequestInvalidClusterValidationLabelsTotalCounter(reg prometheus.Registe
 		},
 	}, []string{methodLabel})
 }
+
+func NewInvalidClusterValidationReporter(cluster string, invalidClusterValidations *prometheus.CounterVec, logger log.Logger) middleware.InvalidClusterValidationReporter {
+	return func(msg string, method string) {
+		level.Warn(logger).Log("msg", msg, "method", method, "clusterValidationLabel", cluster)
```

Comment:
Our convention is snake case in for log message keys:
```suggestion
		level.Warn(logger).Log("msg", msg, "method", method, "cluster_validation_label", cluster)
```